cmd/entrypoints/migrate.go

```diff
@@ -44,15 +45,18 @@ var migrateCmd = &cobra.Command{
 			Password:     databaseConfig.Password,
 			ExtraOptions: databaseConfig.ExtraOptions,
 		}, migrateScope)
-		db, err := gorm.Open(postgresConfigProvider.GetType(), postgresConfigProvider.GetArgs())
+		postgres.New(postgres.Config{
+
+		})
+		db, err := gorm.Open(postgres.Open(postgresConfigProvider.GetArgs()), &gorm.Config{})
 		if err != nil {
 			logger.Fatal(ctx, err)
 		}
-		defer db.Close()
-		db.LogMode(true)
-		if err = db.DB().Ping(); err != nil {
-			logger.Fatal(ctx, err)
-		}
+		//defer db.Close()
+		//db.LogMode(true)
```

Comment:
why is the call to `LogMode(true)` commented out? it should work in the latest gorm too

Reply:
Fixed it.